File: uri/python/1101.py
M = 1
N = 1
soma = 0

while True:
    
    M, N = [int(x) for x in input().split()]
    
    if M > N:
        M, N = N, M
    if M <= 0 or N <= 0:
        break

    for i in range(M, N+1):
        soma += i
    print('{}'.format(str(list(range(M, N+1)))[1:-1].replace(',', '')), 'Sum={}'.format(soma))
    soma = 0

File: uri/c/1101.c
#include <stdio.h>

int main() {

    int M, N, tmp, soma = 0;

    while(1) {

        scanf("%d %d", &M, &N);
        if (M <= 0 || N <= 0) {
            break;
        }

        if (M > N) {
            tmp = M;
            M = N;
            N = tmp;
        }

        for (M; M <= N; M++) {
            printf("%d ", M);
            soma += M;
        }

        printf("Sum=%d\n", soma);
        soma = 0;

    }

    return 0;

}